src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectSettingsController.java

```diff
@@ -44,44 +41,142 @@
 	private final ProjectService projectService;
 	private final ProjectRemoteService projectRemoteService;
 	private final UserService userService;
+	private AnalysisSubmissionService analysisSubmissionService;
+	private IridaWorkflowsService workflowsService;
 
 	public static final String ACTIVE_NAV_SETTINGS = "settings";
 
 	@Autowired
 	public ProjectSettingsController(MessageSource messageSource, MetadataTemplateService metadataTemplateService,
 			ProjectControllerUtils projectControllerUtils, ProjectService projectService,
-			ProjectRemoteService projectRemoteService, UserService userService) {
```

Comment:
We need to clean up this controller and possibly break it into parts, one for each section on the page.  This list of `autowired` dependencies is getting a bit much.

Reply:
Split them up in e8c5deb